src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,57 +70,176 @@ contract DssSpellCollateralAction {
-        //     CollateralOpts({
-        //         ilk:                   ,
-        //         gem:                   ,
-        //         join:                  ,
-        //         clip:                  ,
-        //         calc:                  ,
-        //         pip:                   ,
-        //         isLiquidatable:        ,
-        //         isOSM:                 ,
-        //         whitelistOSM:          ,
-        //         ilkDebtCeiling:        ,
-        //         minVaultAmount:        ,
-        //         maxLiquidationAmount:  ,
-        //         liquidationPenalty:    ,
-        //         ilkStabilityFee:       ,
-        //         startingPriceFactor:   ,
-        //         breakerTolerance:      ,
-        //         auctionDuration:       ,
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
-        // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    uint256 internal constant ZERO_PCT_RATE                  = 1000000000000000000000000000;
+
+    // --- Math ---
+    uint256 internal constant WAD = 10**18;
+
+    // -- RWA007 MIP21 components --
+    address internal constant RWA007                         = 0xD063270642ff718DA1c58E12BD6a2598f7e874B3;
+    address internal constant MCD_JOIN_RWA007_A              = 0x9C9E33E22b683F789411288497f8DC560f1F0466;
+    address internal constant RWA007_A_URN                   = 0xa1b1D392fCB99F8B39c7530a599bCfcd2f1fB22f;
+    address internal constant RWA007_A_JAR                   = 0x708bC8bF869c336ab6f04cf6A62a86a8DFc5f7c4;
+    // Goerli: Coinbase / Mainnet: Coinbase
+    address internal constant RWA007_A_OUTPUT_CONDUIT        = 0x87EaB54D118529Eb15a4286b8A96455ECBdbFD27;
+    // Jar and URN Input Conduits
+    address internal constant RWA007_A_INPUT_CONDUIT_URN     = 0x1C3faBF61B470B0e9aA4Ca5F1A08fcf44ADAb414;
+    address internal constant RWA007_A_INPUT_CONDUIT_JAR     = 0xA7ae4F30f237BB8E8975d22eD777778202F64c91;
+
+    // MIP21_LIQUIDATION_ORACLE params
+
+    // https://gateway.pinata.cloud/ipfs/QmRLwB7Ty3ywSzq17GdDdwHvsZGwBg79oUTpSTJGtodToY
+    string  internal constant RWA007_DOC                     = "QmRLwB7Ty3ywSzq17GdDdwHvsZGwBg79oUTpSTJGtodToY";
```

Comment:
the rwa007 doc hash doesn't seem to have been included to the exec doc or copy, not sure where to check it against.

Reply:
You can find it there: https://forum.makerdao.com/t/rwa007-mip65-monetalis-clydesdale-ces-domain-team-assessment/17787/10?u=m_c